docs/storage.md

```diff
@@ -13,16 +13,16 @@ spec:
       type: openebs_local_storage
 ```
 
-The cluster wil have two storage classes available for you to use:
+The cluster will have two storage classes available for you to use:
 
 ```shell
-bash-5.1# k0s kc get sc
+bash-5.1# k0s kubectl get storageclass
```

Comment:
It's impossible to do this in github, but you get the drift:

```suggestion
k0s kubectl get storageclass
\```
\```shell
```

(remove the prompt, isolate the command into its own box so it can be single-click copied)

Reply:
did like that here and in other place, if I understood correctly